sdk/src/dlob/DLOB.ts

```diff
@@ -545,14 +545,6 @@ export class DLOB {
 			marketType
 		);
 
-		// for spot, multiple makers isn't supported, so don't merge
```

Comment:
shouldn't remove this for spot filler since a fill cannot be both off-drift and on-drift

Reply:
https://github.com/drift-labs/protocol-v2/blob/c9593b26317369918105f2eabf59f6a39287eb18/programs/drift/src/instructions/keeper.rs#L211-L242

Reply:
nevermind, filler should act on if there are makers before attempting to off drift